openpgp/keys.go

```diff
@@ -256,6 +256,46 @@ func (e *Entity) Revoked(now time.Time) bool {
 	return revoked(e.Revocations, now)
 }
 
+// EncryptPrivateKeys encrypts all non-encrypted keys in the entity with the same key
+// derived from the provided passphrase. Public keys and dummy keys are ignored,
+// and don't cause an error to be returned.
+func (e *Entity) EncryptPrivateKeys(passphrase []byte, config *packet.Config) (err error) {
```

Comment:
(and then you don't need to specify the name of the return value)
```suggestion
func (e *Entity) EncryptPrivateKeys(passphrase []byte, config *packet.Config) error {
```